modules/custom/activity_creator/src/ActivityNotifications.php

```diff
@@ -79,16 +100,20 @@ public function markAllNotificationsAsSeen(AccountInterface $account) {
    *   Account object.
    * @param \Drupal\Core\Entity\EntityBase $entity
    *   Entity object.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
    */
   public function markEntityNotificationsAsRead(AccountInterface $account, EntityBase $entity) {
 
     // Retrieve all the activities referring this entity for this account.
-    $ids = $this->getNotificationIds($account, [ACTIVITY_STATUS_RECEIVED, ACTIVITY_STATUS_SEEN], $entity);
+    $entity_query = $this->entityTypeManager->getStorage('activity')->getQuery();
```

Comment:
When I install the branch locally I get an immediate fatal error on this line. The `entityTypeManager` is a dependency but not injected. Something we forgot to do?

Reply:
It should have come from controllerbase abstracter class. I will replace with entityTypeManager() getter function.